plugins/content_distribution/providers/comcast_mrss/lib/ComcastMrssFeed.php

```diff
@@ -300,6 +313,22 @@ public function setThumbAssets(DOMElement $item, array $thumbAssets)
 		}
 	}
 	
+	public function setCaptionAssets (DOMElement $item, array $captionAssets)
+	{
+		foreach ($captionAssets as $captionAsset)
+		{
+			/* @var $captionAsset captionAsset */
+			$content = $this->caption->cloneNode(true);
+			$mediaGroup = $this->xpath->query('media:group', $item)->item(0);
+			$mediaGroup->appendChild($content);
+			$url = $captionAsset->getDownloadUrl(true);
```

Comment:
After checking, I see that the KS expiry is exactly the same as the feed cache expiry - 24 h.